src/Resources/Resources/help/Set-AzManagementGroupDeploymentStack.md

```diff
@@ -141,6 +141,16 @@ Set-AzManagementGroupDeploymentStack -Name <String> -ManagementGroupId <String>
  [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
+### ByParameterFileWithNoTemplate
+```
+Set-AzManagementGroupDeploymentStack -Name <String> -ManagementGroupId <String>
+ -DeploymentSubscriptionId <String> -Location <String> [-Description <String>] [-DeleteAll] [-DeleteResources]
```

Comment:
Please add an example for each cmdlet at least if one parameter set is added.

Reply:
Done!